packages/taquito-core/src/errors.ts

```diff
@@ -264,3 +264,15 @@ export class ProhibitedActionError extends UnsupportedActionError {
     this.name = 'ProhibitedActionError';
   }
 }
+
+/**
+ *  @category Error
+ *  @description Error indicates a failure in grabbing the public key
+ */
+export class PublicKeyNotFoundError extends TaquitoError {
+  constructor(public pkh: string, public cause?: any) {
+    super();
+    this.name = 'KeyNotFoundError';
```

Comment:
The `name` field is different from the class name

Reply:
good catch, will udpate